src/main/java/uk/gov/hmcts/sptribs/common/notification/DecisionIssuedNotification.java

```diff
@@ -133,6 +133,7 @@ private Map<String, String> getUploadedDocuments(CaseData caseData) {
 
         int count = 0;
         if (caseIssueDecision.getDecisionNotice() == NoticeOption.UPLOAD_FROM_COMPUTER) {
+            count++;
```

Comment:
Hope the automated testing will be completed as part of https://github.com/hmcts/sptribs-case-api/pull/1707/files#r1537369471